### ISSUE TITLE
A panic in d2common/d2data/d2datadict.LoadLevelTypes()

### ISSUE THREAD

**Issue by Heimdell** [OpenDiablo2/OpenDiablo2#498]
**Describe the bug**
Game doesn't start on my machine.

**To Reproduce**
1. Build.
2. Run `./OpenDiablo2` to generate a config.
1. Remove `d2x....mpq` s from the config (as I don't have ones).
2. Run `./OpenDiablo2`.

**Expected behavior**
I expected the game to start.

**Screenshots**
![Screenshot from 2020-06-30 11-08-11](https://user-images.githubusercontent.com/1784505/86095534-19a73900-bac3-11ea-9fc7-05803b6e35c3.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 18.04

**Additional context**
My `Patch_D2.mpq` is a modded one; however, its removal changes nothing.

**Comment by essial** [comment]
LevelTypes format tends to change from version to version. Currently our loader only supports the latest patched version of D2. I'm not against supporting other versions of the file, though it is not a priority for us at the moment. I'll leave this issue open in case someone wants to pick this up.

I guess we could look at the layout and set column indexes dynamically.

**Comment by stephenhoran** [comment]
Do we have mpq files from multiple versions to compare layouts?

**Comment by Ziemas** [comment]
Patch_D2.mpq is the only one that changes between patches. At the bottom http://plugy.free.fr/en/index.html there are extracted patches available for each version.

**Comment by stephenhoran** [comment]
Yes good point... but we shouldn't out of bounds on the lvltype.txt file (or really ever). Not supporting other patch's at this time is probably the way to go but I can fix the out of bounds issue.

**Comment by gravestench** [comment]
Any updates on this issue?

@Heimdell are you still getting an error?

**Comment by Heimdell** [comment]
No, it does not appear anymore, you can close. The game runs and looks nice :)
